packages/@webex/internal-plugin-voicea/src/voicea.ts

```diff
@@ -402,34 +411,42 @@ export class VoiceaChannel extends WebexPlugin implements IVoiceaChannel {
 
   /**
    * Turn on Captions
+   * @param {string} spokenLanguage - spoken language code
    * @returns {Promise}
    */
-  public turnOnCaptions = async (): undefined | Promise<void> => {
+  public turnOnCaptions = async (spokenLanguage?): undefined | Promise<void> => {
     if (this.isCaptionProcessing()) return undefined;
     // @ts-ignore
     if (!this.webex.internal.llm.isConnected()) {
       throw new Error('can not turn on captions before llm connected');
     }
 
-    return this.requestTurnOnCaptions();
+    return this.requestTurnOnCaptions(spokenLanguage);
   };
 
   /**
    * Toggle transcribing for highlights
    * @param {bool} activate if true transcribing is turned on
+   * @param {string} spokenLanguage language code for spoken language
    * @returns {Promise}
    */
-  public toggleTranscribing = async (activate: boolean): undefined | Promise<void> => {
+  public toggleTranscribing = async (
```

Comment:
do we need async in this line?

Reply:
We are returning the promise. We don't need it. Also we do not do await anywhere. Will remove it too